src/sentry/eventstream/types.py

```diff
@@ -0,0 +1,8 @@
+class EventStreamEventType:
```

Comment:
do you need to change it for any specific reason? i think it's more safe as an enum like it was originally

Reply:
I think the reason was related to celery task pickling. We could put the string version of the enum values into task messages though.